pkg/apis/policy/v1alpha1/types_certificaterequestpolicy.go

```diff
@@ -53,95 +53,96 @@ type CertificateRequestPolicyList struct {
+	// but _not more_, i.e. a CertificateRequest can request a subset of what
+	// is declared as allowed by the policy.
+	// Omitted fields declares that the equivalent CertificateRequest
+	// field _must_ be omitted or have an empty value for the request to be
+	// permitted.
 	// +optional
 	Allowed *CertificateRequestPolicyAllowed `json:"allowed,omitempty"`
 
-	// Constraints is the set of attributes that _must_ be satisfied by the
-	// CertificateRequest for the request to be permissible by the policy. Empty
-	// or `nil` constraint fields mean CertificateRequests satisfy that field
-	// with any value of their corresponding attribute.
+	// Constraints define fields that _must_ be satisfied by the
+	// CertificateRequest for the request to be allowed by this policy.
+	// Omitted fields will be satisfied by any value on the corresponding
+	// attribute on the request.
 	// +optional
 	Constraints *CertificateRequestPolicyConstraints `json:"constraints,omitempty"`
 
-	// Plugins define a set of plugins and their configuration that should be
-	// executed when this policy is evaluated against a CertificateRequest. A
-	// plugin must already be built within approver-policy for it to be
+	// Plugins define plugins and their configuration that should be
+	// executed when this policy is evaluated against a CertificateRequest.
+	// A plugin must already be built within approver-policy for it to be
 	// available.
 	// +optional
 	Plugins map[string]CertificateRequestPolicyPluginData `json:"plugins,omitempty"`
 
 	// Selector is used for selecting over which CertificateRequests this
-	// CertificateRequestPolicy is appropriate for and so will used for its
+	// CertificateRequestPolicy is appropriate for and so will be used for its
 	// approval evaluation.
 	Selector CertificateRequestPolicySelector `json:"selector"`
 }
 
-// CertificateRequestPolicyAllowed is a set of attributes that are declared as
-// permissible for a CertificateRequest to have those values present. It is
-// permissible for a CertificateRequest to request _less_ than what is allowed,
-// but _not more_, i.e. it is permissible for a CertificateRequest to request a
-// subset of what is allowed.
-// Empty fields or `nil` values declares that the equivalent CertificateRequest
-// field _must_ be omitted or empty for the request to be permitted.
+// CertificateRequestPolicyAllowed defines the attributes a CertificateRequest
+// is allowed to have.
+// A CertificateRequest can request _less_ than what is allowed,
+// but _not more_, i.e. a CertificateRequest can request a subset of what is
+// declared as allowed  by the policy.
+// Omitted fields declares that the equivalent CertificateRequest field _must_
+// be omitted or have an empty value for the request to be permitted.
 type CertificateRequestPolicyAllowed struct {
-	// CommonName defines the X.509 Common Name that is permissible.
+	// CommonName defines the X.509 Common Name a CertificateRequest is
+	// allowed to have.
```

Comment:
This feels wrong, presumably it's the format or a pattern? It can't just be a single value. (But otherwise, the style I've outlined above...)

Reply:
A policy can literally accept just a single common name value. But the field supports wildcards. Not sure about the style you are referring to, sorry. Can you please elaborate?

Reply:
Reworded.